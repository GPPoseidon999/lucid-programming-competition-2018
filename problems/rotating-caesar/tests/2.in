abc 123 THE CODE'S ALIVE!!!