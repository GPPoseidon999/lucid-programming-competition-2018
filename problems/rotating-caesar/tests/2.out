dfh 123 ZOM LYOQ'F OAYMW!!!